iree/compiler/Dialect/HAL/IR/HALOps.td

```diff
@@ -1207,14 +1207,54 @@ def HAL_CommandBufferDispatchOp : HAL_Op<"command_buffer.dispatch"> {
       Value executable, IREE::HAL::ExecutableEntryPointOp entryPoint,
       Value workgroupX, Value workgroupY, Value workgroupZ
     }]>,
-    OpBuilder<[{
-      Builder &builder, OperationState &state, Value commandBuffer,
-      Value executable, unsigned entryPointOrdinal,
-      Value workgroupX, Value workgroupY, Value workgroupZ
-    }]>,
   ];
 }
 
+// TODO(scotttodd): Re-evaluate naming
```

Comment:
Will rename to `hal.command_buffer.dispatch.symbol` and `hal.command_buffer.dispatch` (ordinal).